fingerprint/connect/src/main/java/com/simprints/fingerprint/connect/screens/controller/ConnectScannerControllerFragment.kt

```diff
@@ -68,15 +71,20 @@ internal class ConnectScannerControllerFragment : Fragment(R.layout.fragment_con
     ) { permissions: Map<String, Boolean> ->
         val permission = permissions
             .filterKeys(bluetoothPermissions::contains)
-            .map { (permission, isGranted) -> requireActivity().permissionFromResult(permission, isGranted) }
+            .map { (permission, isGranted) ->
```

Comment:
This is just a styling nitpick, but the old line was significantly better in terms of readability, IMO.